src/main/resources/com/google/api/codegen/java/standalone_sample.snip

```diff
@@ -176,12 +176,15 @@
 
 @private asyncOperationMethodSampleCode(apiMethod, sample)
   {@initCode(sample.sampleInitCode)}
+  OperationFuture<{@apiMethod.responseTypeName}, {@apiMethod.operationMethod.metadataTypeName}> \
+    future = {@apiMethod.apiVariableName}.{@apiMethod.name}(\
```

Comment:
@beccasaurus, for the variable name `future`, I feel I typically see `operation` used more. Thoughts?

Reply:
A quick search in java-docs-samples seems to give me more `future` than `operation` for LRO methods of Gapic clients.

Reply:
Many uses have been abbreviated to just `op` or some use `response` (which I think we shouldn't use.

Reply:
The GAPIC repo seems to use `future` almost exclusively.

Looking at java-docs-samples, I see op, operation, future, response

Let's not do response or op

Operation vs Future?

Well, it's an instance of `OperationFuture` lol. Both!

I could go either way, I'd say `future` because it's consistently used in the library. But `operation` would be fine and consistent with most languages!

Reply:
@nnegrey @beccasaurus 
If y'all agree with `future` can either of you approve this PR?
If there is still a strong preference of `operation` please let me know too :)